docs/source/app_development/widgets.rst

```diff
@@ -1,106 +1,91 @@
-For a simple example, one can use a slider to control the value of a variable.
+ * prepare the input for a calculation (e.g. an atomic structure)
+ * select computational resources and submit a calculation to AiiDA
+ * monitor a running calculation
+ * find and analyze the results of a calculation
 
-.. code-block:: python
-
-    from ipywidgets import FloatSlider
-
-    w = FloatSlider(value=0.0, min=0.0, max=10.0, description="b:");
-    a = 0.0;
-
-    def slider_change(change):
-        global a
+The AiiDAlab widgets help with these common tasks and are preinstalled in the AiiDAlab environment.
+Please see `https://aiidalab-widgets-base.readthedocs.io <https://aiidalab-widgets-base.readthedocs.io/>`_ for documentation on the individual widgets.
 
-        print(change["old"])
-        print(change["new"])
+.. image:: ./include/aiidalab-widgets-base.gif
+    :width: 600px
+    :align: center
+    :alt: text
 
-        a = a + w.value
 
-    a.obersve(slider_change, names="value")
+.. _develop-apps:widgets:more-widgets:
 
-In this example, the ``slider_change`` function will be called each time the slider' value being updated.
+************
+More widgets
+************
 
-Traitlets
-=================
+* `ipywidgets`_: basic GUI components, such as a text, sliders, buttons, progress bars, drowdowns, etc.
+* `widget-periodictable <https://github.com/osscar-org/widget-periodictable>`_ : interactive periodic table for element selection
+* `widget-bandsplot <https://github.com/osscar-org/widget-bandsplot>`_ : plot electronic bandstructure and density of states
+* `widget-jsmol <https://github.com/osscar-org/widget-jsmol>`_ : use the JSmol molecular visualizer inside a Jupyter notebook
 
-In ipywidgets, all the widgets are using the `traitlets <https://traitlets.readthedocs.io/>`_ types for the parameters.
-As shown in the example above, the value of the ``FloatSlider`` is a ``Float`` trait type.
-One can not only show the current value ``value["new"]`` but also the old value ``value["old"]``.
-Traits can emit change events when the attributes are updated.
-This is very useful for creating custom widgets.
 
-Custom widgets
-==============
+**************************
+Terminology and background
+**************************
 
-Ipywidgets package also supports developers in creating their own widgets.
-There is a detailed `tutorial <https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20Custom.html>`_ to show how to develop a custom widget.
-In the ipywidgets framework is based on the Comm framework.
-In the Comm framework, the change events of the traits leads to send a json massage to the Jupyter kernel.
-Then, the frontend will update accordingly.
+Widgets and traitlets
+======================
 
-Develop the custom widgets mainly involve two parts:
+Widgets are eventful Python objects that display GUI components, such as a sliders, buttons, progress bars, drowdowns, etc.
```

Comment:
"Eventful", or "stateful"? I actually never heard the former term. :/

Reply:
This adjective was already there before; I'll just remove it

Reply:
I figured, but wanted to raise that point regardless.